data/biocache/config/biocache-config.properties

```diff
@@ -0,0 +1,233 @@
+######################################################################
```

Comment:
I'm not sure why this file has been added - is it for integration/functional tests? Our Ansible scripts install this file when we deploy the software so just checking if this is duplicating that functionality.

Reply:
I added this file because it is needed to run the webapp locally (via tomcat) and wasn't provided in the repo. The integration/functional tests config is located at `src/test/resources/biocache-test-config.properties`
The file does need to be located outside the repository directory structure so I not sure where is a good place commit it, other repo I've looked at have a `config/` directory off the repos root for dev config files. I've actually created the link from the file system root `/data/` to the repos `data/` directory in order to run the app. Since the config is not specific to the dev environment that seems to work. 
Haven't updated the dev installation instructions (PDF) though.

Reply:
@djtfmartin comments/suggestions?

Reply:
After discussing with Dave, we suggest the config file be removed from the code and be put in a [GH wiki page](https://github.com/AtlasOfLivingAustralia/biocache-service/wiki) (linked from main README.md). Might be worth adding a note on the page about not including sensitive values such as passwords and api keys for anyone updating the file.

Reply:
I've ported the "Biocache Service Starter Pack-v7-20200415_111740.pdf" content into the wiki and updated documentation for tomcat 9. Added the config files and made reference link from README.md